src/app/games/coerceo/tests/coerceo.component.spec.ts

```diff
@@ -127,7 +127,7 @@ describe('CoerceoComponent', () => {
         }));
     });
     describe('First click', () => {
-        it('should accept tiles exchange proposal as first click', fakeAsync(async() => {
+        it('should refuse tiles exchange proposal as first click', fakeAsync(async() => {
```

Comment:
hmm, le "as first click" dans le it est pourritte non ? C'est refusé parce qu'il y a pas assez de tuile, pas parce que c'est ton premier click !

Reply:
Saint-Severin